fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordStoreBase.java

```diff
@@ -720,7 +720,23 @@ default CompletableFuture<FDBStoredRecord<M>> loadRecordAsync(@Nonnull final Tup
      */
     @Nonnull
     @API(API.Status.EXPERIMENTAL)
-    CompletableFuture<FDBSyntheticRecord> loadSyntheticRecord(@Nonnull Tuple primaryKey);
+    default CompletableFuture<FDBSyntheticRecord> loadSyntheticRecord(@Nonnull Tuple primaryKey) {
+        return loadSyntheticRecord(primaryKey, IndexOrphanBehavior.ERROR);
+    }
+
+    /**
+     * Load a {@link FDBSyntheticRecord synthetic record} by loading its stored constituent records and synthesizing it from them.
+     * In case any of the constituents are missing, the following will be returned:
+     * - ERROR: Exception is thrown
+     * - SKIP: Future(null) is returned
+     * - RETURN: Future(Record with no constituents) is returned
```

Comment:
This won't render well in our javadoc, better to be a `<ul>`, and use links for the behaviors, I think:
```suggestion
     * <ul>
     *     <li>{@link IndexOrphanBehavior#ERROR}: Exception is thrown</li>
     *     <li>{@link IndexOrphanBehavior#SKIP}: Future(null) is returned</li>
     *     <li>{@link IndexOrphanBehavior#RETURN}: Future(Record with no constituents) is returned</li>
     * </ul>
```

Reply:
Done.